opsdroid/parsers/dialogflow.py

```diff
@@ -5,7 +5,7 @@
 
 import aiohttp
 
-from opsdroid.const import DEFAULT_LANGUAGE
+from opsdroid.const import DEFAULT_LANGUAGE, DIALOGFLOW_API_ENDPOINT, DIALOGFLOW_API_VERSION
```

Comment:
This line is also too long. Please break over multiple lines.

Reply:
Okay I am doing same.